apps/sms/locales/sms.en-US.properties

```diff
@@ -316,6 +316,39 @@ message-delivery-status-delivered.ariaLabel          = Delivered
 message-delivery-status-read.ariaLabel               = Read
 message-delivery-status-error.ariaLabel              = Error
 
+# Late arrival notices
+late-arrival-notice-in-months         = {[ plural(lateness) ]}
+late-arrival-notice-in-months[zero]   = 
+late-arrival-notice-in-months[one]    = {{ lateness }}mth late
+late-arrival-notice-in-months[other]  = {{ lateness }}mths late
+late-arrival-notice-in-days           = {{ lateness }}d late
+late-arrival-notice-in-hours          = {{ lateness }}h late
+late-arrival-notice-in-minutes        = {{ lateness }}m late
+
+# Lateness message
+message-lateness                = Lateness:
+
+message-lateness-months         = {[ plural(months) ]}
+message-lateness-months[zero]   =
+message-lateness-months[one]    = {{ months }}mth
+message-lateness-months[other]  = {{ months }}mths
+
+message-lateness-days           = {[ plural(days) ]}
+message-lateness-days[zero]     = 
+message-lateness-days[other]    = {{ days }}d
+
+message-lateness-hours          = {[ plural(hours) ]}
+message-lateness-hours[zero]    =
+message-lateness-hours[other]   = {{ hours }}h
+
+message-lateness-minutes        = {[ plural(minutes) ]}
+message-lateness-minutes[zero]  =
+message-lateness-minutes[other] = {{ minutes }}m
+
+message-lateness-seconds        = {[ plural(seconds) ]}
+message-lateness-seconds[zero]  =
+message-lateness-seconds[other] = {{ seconds }}s
+
```

Comment:
stéphane, maybe Zibi will contradict me here, but you need to put all "flavors" of plurals here, even if in english we don't use all of them; here is an example:
```
selected-threads         = {[ plural(n) ]}
selected-threads[zero]   = none selected
selected-threads[one]    = {{n}} selected
selected-threads[two]    = {{n}} selected
selected-threads[few]    = {{n}} selected
selected-threads[many]   = {{n}} selected
selected-threads[other]  = {{n}} selected
```

I think this is a limitation of our translation tools.

Also we need to provide a translation for "zero"; if we want nothing to be displayed, then we need to do it programmatically (either hide in CSS or not add it in JS).